pkg/libvmi/storage.go

```diff
@@ -219,15 +228,20 @@ func newVolume(name string) v1.Volume {
 	return v1.Volume{Name: name}
 }
 
-func newContainerVolume(name, image string) v1.Volume {
-	return v1.Volume{
+func newContainerVolume(name, image string, imagePullPolicy ...k8sv1.PullPolicy) v1.Volume {
```

Comment:
```suggestion
func newContainerVolume(name, image string, imagePullPolicy k8sv1.PullPolicy) v1.Volume {
```
In this way you can directly check `imagePullPolicy != ""` later and prevent multiple imagePullPolices to be passed